src/applications/static-pages/facilities/FacilityAppointmentWaitTimesWidget.jsx

```diff
@@ -6,19 +6,22 @@ import { connect } from 'react-redux';
 import _ from 'lodash';
 
 export class FacilityAppointmentWaitTimesWidget extends React.Component {
-  /* @param waitTimes (object) - an object with a list of keys of facility services names in camelCase as keys with a value of an object
-  *  @param service (string) - a health service taxonomy such as "Primary Care"
-  *  @param established (boolean - default = false) - which object value is desired. two choices: 'new' or 'established'. true gives 'established' value. false gives 'new' value.
-  *  returns a string formatted with the number of days for appointment wait times ex. '8 days'
-  */
-  appointmentWaitTime(waitTimes, service, established = false) {
-    // convert service to camel case
-    const serviceKey = _.camelCase(service);
-    // find service in waitTimes object as a key
-    const time = waitTimes[serviceKey];
-    // return waitTimes for service
-    const waitTime = Number(time[established ? 'established' : 'new']);
-    return waitTime ? `${waitTime.toFixed(0)} days` : '';
+  appointmentWaitTime(waitTime, service, established = false) {
+    return (
+      <div className="facility-satisfaction-tile vads-u-background-color--gray-lightest vads-u-padding-x--2 vads-u-padding-top--1 vads-u-padding-bottom--1p5">
+        <p className="vads-u-margin--0">
+          {established ? 'Existing patient' : 'New patient'}
+        </p>
+        <p
+          id={`facility-${_.camelCase(service)}-${
+            established ? 'existing' : 'new'
+          }-patient-wait-time`}
+          className="vads-u-font-size--lg vads-u-font-weight--bold vads-u-margin--0 vads-u-font-family--serif"
+        >
+          {`${waitTime.toFixed(0)} days`}
```

Comment:
Are we still coercing the `waitTime` into a number? If not, this line will throw an error.

Reply:
I've seen the API return the wait times as strings, so it's a good idea IMO to be defensive and coerce the value to a Number before calling `toFixed`.